src/app/page.tsx

```
@@ -1,11 +1,21 @@
 
 export default function HomePage() {
     return (
+        <div>
+
         <main className="flex min-h-screen flex-col items-center justify-center bg-gradient-to-b from-[#2e026d] to-[#15162c] text-white">
             <div className="container flex flex-col items-center justify-center gap-12 px-4 py-16">
                 <h1 className="text-6xl font-bold">GeFighting</h1>
             </div>
+
+            <Link href="/about">
+                        <button className="px-4 py-2 bg-blue-500 text-white rounded-md">Go to About</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)